Cards/archieve.cpp

```diff
@@ -7044,3 +7053,253 @@ Parasite::Parasite()
+                || enemy.rowEffect(Seige) == ImpenetrableFogEffect
+                || ally.rowEffect(Meele) == ImpenetrableFogEffect
+                || ally.rowEffect(Range) == ImpenetrableFogEffect
+                || ally.rowEffect(Seige) == ImpenetrableFogEffect)
+            boost(this, 1, ally, enemy, this);
+    };
+}
+
+Draug::Draug()
+{
+    id = "132101";
+    name = "Draug";
+    text = "Resurrect units as 1-power Draugirs until you fill this row.";
+    url = "https://gwent.one/image/card/low/cid/png/" + id + ".png";
+    power = powerBase = 10;
+    rarity = Gold;
+    faction = Monster;
+    tags = { Cursed, Officer };
+
+    _onDeploy = [=](Field &ally, Field &enemy) {
+        const Row row = _findRowAndPos(this, ally).row();
+        const std::vector<Card *> cards = cardsFiltered(ally, enemy, {isUnit}, AllyDiscard);
+        for (Card *card : cards) {
+            if (isRowFull(ally.row(row)))
+                return;
+            moveExistedUnitToPos(card, rowAndPosLastInExactRow(ally, row), ally, enemy, this);
+            transform(card, Draug::Draugir(), ally, enemy, this);
+        }
+    };
+}
+
+Draug::Draugir::Draugir()
+{
+    // TODO: find picture and verify stats
+    id = "132101";
+    name = "Draugir";
+    text = "";
+    url = "https://gwent.one/image/card/low/cid/png/" + id + ".png";
+    power = powerBase = 1;
+    rarity = Bronze;
+    faction = Monster;
+    tags = { Cursed };
+    isDoomed = true;
+}
+
+CelaenoHarpy::CelaenoHarpy()
+{
+    id = "132217";
+    name = "Celaeno Harpy";
+    text = "Spawn 2 Harpy Eggs to the left of this unit.";
+    url = "https://gwent.one/image/card/low/cid/png/" + id + ".png";
+    power = powerBase = 6;
+    rarity = Bronze;
+    faction = Monster;
+    tags = { Beast };
+
+    _onDeploy = [=](Field &ally, Field &enemy) {
+        spawnNewUnitToPos(new HarpyEgg(), rowAndPosToTheLeft(this, ally, 1), ally, enemy, this);
+        spawnNewUnitToPos(new HarpyEgg(), rowAndPosToTheLeft(this, ally, 1), ally, enemy, this);
+    };
+}
+
+CelaenoHarpy::HarpyEgg::HarpyEgg()
+{
+    id = "132316";
+    name = "Harpy Egg";
+    text = "When Consumed, boost Consuming unit by 4. Deathwish: Spawn a Harpy Hatchling on a random row.";
+    url = "https://gwent.one/image/card/low/cid/png/" + id + ".png";
+    power = powerBase = 1;
+    rarity = Bronze;
+    faction = Monster;
+    isDoomed = true;
+
+    _onConsumed = [=](Field &ally, Field &enemy, Card *src) {
+        boost(src, 4, ally, enemy, this);
+    };
+}
+
+ArachasBehemoth::ArachasBehemoth()
+{
+    id = "132201";
+    name = "Arachas Behemoth";
+    text = "The next 4 times you Consume a unit, Spawn an Arachas Hatchling on a random row.";
+    url = "https://gwent.one/image/card/low/cid/png/" + id + ".png";
+    power = powerBase = 3;
+    rarity = Bronze;
+    faction = Monster;
+    tags = { Insectoid };
+
+    _onAllyConsume = [=](Field &ally, Field &enemy, Card *) {
+        if(numberOfTimes >= 4 || !isOnBoard(this, ally))
```

Comment:
In original game there was performed with a timer, rather invinsible variable. I suggest to use `setTimer` `tick`